### PR TITLE
Refactor multilanguage implementation

### PR DESCRIPTION
**Pull request by sauloperez**
## Description

I refactored the site to use [jekyll-multiple-languages-plugin](Anthony-Gaudino/jekyll-multiple-languages-plugin). This plugin adds a folder for each language defined in config. As a result we moved all posts to each language folder so that we keep the current behavior of mixing languages.

Now, to add new pages we'll have to add the template plus its translations. That is, `about.html` with a call to `translate_file`, plus the files `_i18n/ca/about.md` and `_i18n/es/about.md`.

Then, all translation keys live in either `_i18n/ca.yml` and `_i18n/es.yml`.

:warning: Something I just realized is that 

> The main administrator @Anthony-Gaudino, has disappeared so we are in a bit of a sad state here. I still have access to the Ruby Gem and can release new versions but I do not use any Jekyll sites and have no way of verifying PRs. In short this project really need a new owner, is that you? Or can you atleast verify all PRs so I can merge and make a release?

Source: https://github.com/Anthony-Gaudino/jekyll-multiple-languages-plugin/pull/125#issuecomment-432645628

After seeing people using home-made solutions and other plugins I can tell this is one seems the best to me. Even, if we need to, we could mess with the code. It's not a big deal.

See more details in: Anthony-Gaudino/jekyll-multiple-languages-plugin

## Fixes

I also took the chance to fix `Seguir leyendo`. We were displaying it in both Catalan and Spanish. Now that's translated.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,6 +9,7 @@ source "https://rubygems.org"
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
 gem "jekyll", "~> 3.7.2"
+gem 'jekyll-multiple-languages-plugin'
```
Comment (danypr92) [inline Gemfile]: We can specify the version, no?
And we can use also the `"` as in the previous line.

```
gem "jekyll-multiple-languages-plugin", "~> 1.6"
```

**Review by danypr92 — APPROVED** [review]

File: _layouts/main.html [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
-  <h2>{{ page.projects_title }}</h2>
+  <h2>{% t page.projects_title %}</h2>
 
   <div class='project'>
     <div class='logo'>
       <img src='{{ "/assets/img/projects/katuma.png" | prepend:site.baseurl }}' width='140' height='140' />
     </div>
     <div class='description'>
-      {{ page.katuma_description | markdownify }}
+      {% t page.katuma_description %}
```
Comment (sauloperez) [inline _layouts/main.html:16]: Sadly, `markdownify` doesn't work with `{% %}` tags so we need to write these translations in HTML instead.

File: Gemfile (hunk 1 above)
Comment (sauloperez) [reply Gemfile]: Nice one!

File: _layouts/main.html (hunk 2 above)
Comment (andresgutgon) [reply _layouts/main.html:16]: What if you markdownify in an `assign` variable and then use it? Would that work?

Comment (sauloperez) [reply _layouts/main.html:16]: I tried and it doesn't. Looks like you can't use filters there either.

Comment (andresgutgon) [reply _layouts/main.html:16]: So do a helper in Ruby that use `markdownify`. I think Jekyll has helpers a la Rails

Comment (sauloperez) [reply _layouts/main.html:16]: Yes, I thought about that. Or even open a PR against that plugin to allow for this but I don't think it's a big deal now. We were just markdownifying couple links and a `<strong>`.

We can create an issue for it in the future though if things get painful.

**Comment by enricostano** [comment]
Is this Jekill plugin supported by GH Pages?

**Comment by sauloperez** [comment]
That's a good point. It just adds extra build steps but I'm not sure how that relates to GH's build process... I bet not very well... :see_no_evil:

**Comment by andresgutgon** [comment]
The build can happen in your machine. It should not be a problem afaik

**Comment by sauloperez** [comment]
We'll need to do https://github.com/coopdevs/coopdevs.github.io/issues/33 first.

**Comment by enricostano** [comment]
> The build can happen in your machine. It should not be a problem afaik

In that case we loose the automated build/deploy on code change, that's the main selling point of GH pages IMHO.

This is why I started to investigate how to support Jekyll deploys on our servers: https://github.com/coopdevs/coopdevs-provisioning/blob/master/roles/coopdevs.org/files/post-receive

It works, but we still don't use it.

**Comment by andresgutgon** [comment]
What about Netlify? It's more flexible than Gh pages

**Comment by andresgutgon** [comment]
Anyway this it's juat build in local machine and push to gh-pages branch. Not hard to implement. Does people change content on this side from GitHub UI? If all the changes are done by a developer I would stick with Gh pages

**Comment by enricostano** [comment]
> What about Netlify? It's more flexible than Gh pages

This is why I stopped investigating our solution, I wanted to try Netlify.... but, you know... time...

>  If all the changes are done by a developer I would stick with Gh pages

We have/want more people than developers to be able to contribute to our projects, that's why I would keep the build and deploy automation in place.

**Comment by sauloperez** [comment]
Up & Running 🎉 !

**Pull request merged**